packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -576,10 +593,11 @@ class VirtualListBasic extends Component {
 
 	getXY = (primaryPosition, secondaryPosition) => (this.isPrimaryDirectionVertical ? {x: secondaryPosition, y: primaryPosition} : {x: primaryPosition, y: secondaryPosition})
 
-	getClientSize = (node) => ({
-		clientWidth: node.clientWidth,
-		clientHeight: node.clientHeight
-	})
+	getContentSize = (props) => {
+		const contentSizes = props.clientSize || props.scrollContentRef.current;
+
+		return contentSizes && props.getContentSize ? props.getContentSize(contentSizes) : contentSizes;
```

Comment:
Don't we need a proptypes for `getContentSize`?

Reply:
No lint warning/error so I missed it.